demo/Cargo.toml

```diff
@@ -54,28 +52,29 @@ ron = "0.6"
 profiling = "1.0.1"
 stats_alloc = { version = "0.1.8", optional = true }
 bevy_tasks = "0.5.0"
+raw-window-handle = "0.3.3"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+env_logger = "0.6"
+winit = { version = "0.24" }
 
 [features]
 default = [
-    "sdl2-bundled",
-    "sdl2-static-link",
     "profile-with-puffin",
-    "egui"
+    "use-egui"
```

Comment:
conventionally it should be possible to use the egui crate name directly to turn on egui related stuff, but we need it to turn on multiple crates, so I think we have to use a feature